packages/testing-application/lib/factories/searchAlgorithms/RandomSearchFactory.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020-2023 Delft University of Technology and SynTest contributors
+ *
+ * This file is part of SynTest Framework - SynTest Core Graphing Plugin.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  Encoding,
+  RandomSearch,
+  SearchAlgorithm,
+  SimpleObjectiveManager,
+} from "@syntest/core";
+import {
+  SearchAlgorithmPlugin,
+  SearchAlgorithmOptions,
+} from "../../plugin/SearchAlgorithmPlugin";
+import { pluginRequiresOptions } from "@syntest/cli";
+/**
+ * Factory plugin for RandomSearch
+ *
+ * @author Dimitri Stallenberg
+ */
+export class RandomSearchFactory<T extends Encoding>
+  implements SearchAlgorithmPlugin<T>
+{
+  name = "RandomSearch";
```

Comment:
This should be called Random

Reply:
Only the name right? not the class name?
Fixed that.

Reply:
Yes, only the name. Not the class. The search already comes from it being a search algorithm.

Reply:
Okay!